### ISSUE TITLE
Saskatoon, SK, Canada data contains coordinates only

### ISSUE THREAD

**Issue by JosephKuchar** [openaddresses/openaddresses#4660]
The saskatoon csv contains lats and lons, but all entries are missing street number, name, etc.

**Comment by iandees** [comment]
It looks like the KML file we download has attributes formatted as HTML in a way that are very hard to parse. I don't suppose you could reach out to the Saskatoon open data folks and ask for updated addresses in a more usable format?

**Comment by JosephKuchar** [comment]
> It looks like the KML file we download has attributes formatted as HTML in a way that are very hard to parse. I don't suppose you could reach out to the Saskatoon open data folks and ask for updated addresses in a more usable format?

Sure, I can give it a shot. I notice the data is available in csv, but without coordinates, so I'll ask them if they can add that information to the csv.

**Comment by JosephKuchar** [comment]
A bit of a late follow-up: I never heard back from the SK open data portal, but I did [write a script](https://github.com/CSBP-CPSE/LODE-ECDO/blob/master/scripts/Addresses/process_saskatoon.py) to parse the kml file this morning.

**Comment by andrewharvey** [comment]
fixed in #4796
